fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/SqlToScalarOperatorTranslator.java

```diff
@@ -287,6 +288,13 @@ public ScalarOperator visit(ParseNode node, Context context) {
             return super.visit(node, context);
 
         }
+        @Override
+        public ScalarOperator visitParameterExpr(Parameter node, Context context) {
+            if (node.getExpr() == null) {
+                throw new SemanticException("Unknown parameter");
+            }
```

Comment:
why will appear null?

Reply:
just add some redundancy checking for not considering factors